2024-08-06 19:57:37,746 INFO    StreamThr :2146132 [internal.py:wandb_internal():86] W&B internal server running at pid: 2146132, started at: 2024-08-06 19:57:37.746569
2024-08-06 19:57:37,748 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status
2024-08-06 19:57:37,748 INFO    WriterThread:2146132 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/run-kazshqy2.wandb
2024-08-06 19:57:37,748 DEBUG   SenderThread:2146132 [sender.py:send():382] send: header
2024-08-06 19:57:37,748 DEBUG   SenderThread:2146132 [sender.py:send():382] send: run
2024-08-06 19:57:37,991 INFO    SenderThread:2146132 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files
2024-08-06 19:57:37,991 INFO    SenderThread:2146132 [sender.py:_start_run_threads():1136] run started: kazshqy2 with start time 1722945457.746405
2024-08-06 19:57:37,995 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 19:57:37,995 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: check_version
2024-08-06 19:57:38,075 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 19:57:38,080 DEBUG   HandlerThread:2146132 [system_info.py:__init__():32] System info init
2024-08-06 19:57:38,080 DEBUG   HandlerThread:2146132 [system_info.py:__init__():47] System info init done
2024-08-06 19:57:38,080 INFO    HandlerThread:2146132 [system_monitor.py:start():194] Starting system monitor
2024-08-06 19:57:38,080 INFO    SystemMonitor:2146132 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 19:57:38,080 INFO    HandlerThread:2146132 [system_monitor.py:probe():214] Collecting system info
2024-08-06 19:57:38,080 INFO    SystemMonitor:2146132 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 19:57:38,081 INFO    SystemMonitor:2146132 [interfaces.py:start():190] Started disk monitoring
2024-08-06 19:57:38,081 INFO    SystemMonitor:2146132 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 19:57:38,082 INFO    SystemMonitor:2146132 [interfaces.py:start():190] Started memory monitoring
2024-08-06 19:57:38,082 INFO    SystemMonitor:2146132 [interfaces.py:start():190] Started network monitoring
2024-08-06 19:57:38,089 DEBUG   HandlerThread:2146132 [system_info.py:probe():196] Probing system
2024-08-06 19:57:38,089 DEBUG   HandlerThread:2146132 [system_info.py:_probe_git():181] Probing git
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_info.py:_probe_git():189] Probing git done
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_info.py:probe():244] Probing system done
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T11:57:38.089182', 'startedAt': '2024-08-06T11:57:37.741519', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 2612.0275, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 3721.484, 'min': 800.0, 'max': 5500.0}, {'current': 3859.281, 'min': 800.0, 'max': 5500.0}, {'current': 3659.187, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3686.145, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3610.816, 'min': 800.0, 'max': 5500.0}, {'current': 3706.457, 'min': 800.0, 'max': 5500.0}, {'current': 3604.136, 'min': 800.0, 'max': 5600.0}, {'current': 3835.203, 'min': 800.0, 'max': 5600.0}, {'current': 3846.736, 'min': 800.0, 'max': 5600.0}, {'current': 3839.455, 'min': 800.0, 'max': 5600.0}, {'current': 3951.658, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3605.019, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3944.257, 'min': 800.0, 'max': 4300.0}, {'current': 3993.541, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 443.37454986572266}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 19:57:38,091 INFO    HandlerThread:2146132 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 19:57:38,091 INFO    HandlerThread:2146132 [system_monitor.py:probe():227] Publishing system info
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 19:57:38,091 DEBUG   HandlerThread:2146132 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 19:57:38,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/requirements.txt
2024-08-06 19:57:38,992 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/conda-environment.yaml
2024-08-06 19:57:39,763 DEBUG   HandlerThread:2146132 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 19:57:39,763 DEBUG   HandlerThread:2146132 [system_info.py:_save_code():90] Saving code
2024-08-06 19:57:39,764 DEBUG   HandlerThread:2146132 [system_info.py:_save_code():111] Saving code done
2024-08-06 19:57:39,764 DEBUG   HandlerThread:2146132 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 19:57:39,767 ERROR   HandlerThread:2146132 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 19:57:39,767 DEBUG   HandlerThread:2146132 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 19:57:39,767 INFO    HandlerThread:2146132 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 19:57:39,768 DEBUG   SenderThread:2146132 [sender.py:send():382] send: files
2024-08-06 19:57:39,768 INFO    SenderThread:2146132 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 19:57:39,768 INFO    SenderThread:2146132 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 19:57:39,770 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 19:57:39,770 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: stop_status
2024-08-06 19:57:39,771 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 19:57:39,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/conda-environment.yaml
2024-08-06 19:57:39,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/code/llama/gmm.py
2024-08-06 19:57:39,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/wandb-metadata.json
2024-08-06 19:57:39,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/code/llama
2024-08-06 19:57:39,991 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/code
2024-08-06 19:57:40,022 DEBUG   SenderThread:2146132 [sender.py:send():382] send: telemetry
2024-08-06 19:57:40,023 DEBUG   SenderThread:2146132 [sender.py:send():382] send: config
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: metric
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: telemetry
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: metric
2024-08-06 19:57:40,025 WARNING SenderThread:2146132 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: telemetry
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: telemetry
2024-08-06 19:57:40,025 DEBUG   SenderThread:2146132 [sender.py:send():382] send: config
2024-08-06 19:57:40,027 DEBUG   SenderThread:2146132 [sender.py:send():382] send: exit
2024-08-06 19:57:40,028 INFO    SenderThread:2146132 [sender.py:send_exit():589] handling exit code: 1
2024-08-06 19:57:40,028 INFO    SenderThread:2146132 [sender.py:send_exit():591] handling runtime: 1
2024-08-06 19:57:40,030 INFO    SenderThread:2146132 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:57:40,030 INFO    SenderThread:2146132 [sender.py:send_exit():597] send defer
2024-08-06 19:57:40,031 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,031 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 19:57:40,031 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,032 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 19:57:40,032 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 1
2024-08-06 19:57:40,032 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,032 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 19:57:40,032 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,033 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 19:57:40,033 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 2
2024-08-06 19:57:40,033 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,033 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 19:57:40,033 INFO    HandlerThread:2146132 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 19:57:40,034 INFO    HandlerThread:2146132 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 19:57:40,034 DEBUG   SystemMonitor:2146132 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 19:57:40,035 DEBUG   SystemMonitor:2146132 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 19:57:40,035 DEBUG   SystemMonitor:2146132 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-06 19:57:40,036 INFO    HandlerThread:2146132 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 19:57:40,097 INFO    HandlerThread:2146132 [interfaces.py:finish():202] Joined gpu monitor
2024-08-06 19:57:40,097 INFO    HandlerThread:2146132 [interfaces.py:finish():202] Joined memory monitor
2024-08-06 19:57:40,098 INFO    HandlerThread:2146132 [interfaces.py:finish():202] Joined network monitor
2024-08-06 19:57:40,098 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 3
2024-08-06 19:57:40,098 DEBUG   SenderThread:2146132 [sender.py:send():382] send: stats
2024-08-06 19:57:40,098 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,098 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-06 19:57:40,098 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 4
2024-08-06 19:57:40,098 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,098 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-06 19:57:40,098 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-06 19:57:40,098 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 5
2024-08-06 19:57:40,098 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,098 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-06 19:57:40,099 DEBUG   SenderThread:2146132 [sender.py:send():382] send: summary
2024-08-06 19:57:40,099 INFO    SenderThread:2146132 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 19:57:40,099 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,099 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-06 19:57:40,099 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 6
2024-08-06 19:57:40,099 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,099 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-06 19:57:40,099 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,099 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-06 19:57:40,101 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:57:40,209 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 7
2024-08-06 19:57:40,209 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:40,209 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-06 19:57:40,209 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:40,209 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-06 19:57:40,217 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:57:40,217 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:40,217 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921C47631173F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:40,575 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:57:40,575 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:40,575 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921C48599A46F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:40,988 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 19:57:40,992 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/config.yaml
2024-08-06 19:57:40,993 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/wandb-summary.json
2024-08-06 19:57:40,993 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/output.log
2024-08-06 19:57:41,504 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:57:41,504 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:41,504 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921C4C5D4E9AE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:41,900 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:57:41,901 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:41,901 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921C4DE4C009B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:41,901 INFO    wandb-upload_0:2146132 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 19:57:42,040 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 8
2024-08-06 19:57:42,040 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 19:57:42,040 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:42,040 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-06 19:57:42,040 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:42,040 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-06 19:57:42,040 INFO    SenderThread:2146132 [job_builder.py:build():298] Attempting to build job artifact
2024-08-06 19:57:42,040 INFO    SenderThread:2146132 [job_builder.py:_get_source_type():439] no source found
2024-08-06 19:57:42,041 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 9
2024-08-06 19:57:42,041 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:42,041 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-06 19:57:42,041 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:42,041 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 9
2024-08-06 19:57:42,041 INFO    SenderThread:2146132 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-06 19:57:42,993 INFO    Thread-12 :2146132 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/output.log
2024-08-06 19:57:42,995 INFO    SenderThread:2146132 [dir_watcher.py:finish():388] scan: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files
2024-08-06 19:57:42,995 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/wandb-summary.json wandb-summary.json
2024-08-06 19:57:42,995 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/requirements.txt requirements.txt
2024-08-06 19:57:43,000 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/output.log output.log
2024-08-06 19:57:43,004 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/config.yaml config.yaml
2024-08-06 19:57:43,008 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/wandb-metadata.json wandb-metadata.json
2024-08-06 19:57:43,009 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/conda-environment.yaml conda-environment.yaml
2024-08-06 19:57:43,016 INFO    SenderThread:2146132 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/code/llama/gmm.py code/llama/gmm.py
2024-08-06 19:57:43,016 INFO    SenderThread:2146132 [sender.py:transition_state():617] send defer: 10
2024-08-06 19:57:43,016 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: defer
2024-08-06 19:57:43,017 INFO    HandlerThread:2146132 [handler.py:handle_request_defer():172] handle defer: 10
2024-08-06 19:57:43,017 DEBUG   SenderThread:2146132 [sender.py:send_request():409] send_request: defer
2024-08-06 19:57:43,017 INFO    SenderThread:2146132 [sender.py:send_request_defer():613] handle sender defer: 10
2024-08-06 19:57:43,017 INFO    SenderThread:2146132 [file_pusher.py:finish():175] shutting down file pusher
2024-08-06 19:57:43,163 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:57:43,165 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:43,167 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921C5299268E7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:43,343 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:57:43,344 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:43,344 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C52E8F2AC7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:43,422 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:57:43,422 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:43,422 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921C5349B3442</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:43,585 INFO    wandb-upload_2:2146132 [upload_job.py:push():131] Uploaded file /home/rwkv/JL/GMM/llama/wandb/run-20240806_195737-kazshqy2/files/wandb-summary.json
2024-08-06 19:57:43,875 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:57:43,875 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:43,875 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921C552C4249C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:44,035 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:57:44,035 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:44,035 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921C55D4CDFC5</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:44,035 INFO    wandb-upload_0:2146132 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 19:57:44,443 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:57:44,443 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:44,444 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C574FE7FBA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:44,552 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:57:44,552 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:44,553 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921C5773AEF39</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:44,913 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:57:44,913 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:44,913 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921C5871721FC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:45,991 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:57:46,776 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:57:46,777 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:46,777 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921C600BB17F7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:46,888 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:57:46,888 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:46,888 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C5F6DD34F1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:47,662 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:57:47,662 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:47,662 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921C6292578FC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:48,018 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:57:48,890 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:57:48,890 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:48,890 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921C67EBB3A7E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:49,003 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:57:49,004 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:49,004 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921C67139EC07</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:50,993 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:57:51,721 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:57:51,722 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:51,722 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C71BF6BDB3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:51,866 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:57:51,866 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:51,866 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921C730077349</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:51,979 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:57:51,980 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:51,980 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921C72D5CC14F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:53,019 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:57:55,995 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:57:57,335 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:57:57,335 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:57,336 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921C87078BB5F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:57:58,020 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:57:58,222 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:57:58,222 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:57:58,222 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921C89B3D10DE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:00,996 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:01,321 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:58:01,321 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:01,321 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C957D5ECAD</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:01,651 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:58:01,651 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:01,651 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921C97739EC2A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:01,683 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:58:01,684 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:01,684 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921C9721F16D6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:03,021 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:03,038 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:03,038 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:03,038 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921C9B8E22DAC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:04,358 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:04,358 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:04,359 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921CA148D9D4D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:05,998 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:06,545 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:06,545 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:06,545 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921CA958716F2</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:08,022 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:11,000 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:11,259 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:11,259 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:11,260 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921CBACAD6219</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:13,023 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:15,431 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115740Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=60b3dc7a32ac242b4adf624065866ce41978acf29119060e37b5c99206212ad0
2024-08-06 19:58:15,431 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5183', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:15,431 ERROR   wandb-upload_0:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E921CCAC983196</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:16,003 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:16,490 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115739Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=598741c96ea924bd8d2a8fe4066459d455892ce062030daa098d66c8998a1fa8
2024-08-06 19:58:16,490 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:16,490 ERROR   wandb-upload_1:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E921CCE6D1707B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:18,024 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:18,415 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=165c04c1f11fd03b5fd8d4940e459c7337e55bf219e36c31f653f4575d64bf0e
2024-08-06 19:58:18,416 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:18,416 ERROR   wandb-upload_3:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E921CD4837DA88</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:18,947 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=85ea8722430a4ad0df69c8f66db6b10e0e43792ba4313a7520c4c5a2340a87d5
2024-08-06 19:58:18,947 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8381', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:18,947 ERROR   wandb-upload_4:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/output.log</Resource><Region>wandb-local</Region><RequestId>17E921CD77A18521</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:19,550 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=af2c6103e5204a1ed508c7100f2d638c38f1df7d86cdaa3827d1936172d77ee5
2024-08-06 19:58:19,550 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:19,551 ERROR   wandb-upload_6:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E921CD9EA66328</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:21,004 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:21,083 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:21,083 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:21,083 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921CDF95050C1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:23,025 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:26,007 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:28,026 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:31,008 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:33,027 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:36,010 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:38,028 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:39,904 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kazshqy2/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T115743Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=838d13390b3d56706e81dd6799b6fda1acf9c2735db75da941dc7f4b9edc32cd
2024-08-06 19:58:39,904 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11185', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 19:58:39,904 ERROR   wandb-upload_5:2146132 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kazshqy2/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kazshqy2/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E921D23C0EFCC1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 19:58:41,012 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:43,029 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 19:58:46,014 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 19:58:48,030 DEBUG   HandlerThread:2146132 [handler.py:handle_request():146] handle_request: status_report
